types/express-session/index.d.ts

```diff
@@ -64,6 +64,7 @@ declare namespace session {
     proxy?: boolean;
     saveUninitialized?: boolean;
     unset?: string;
+    sameSite?: boolean | string;
```

Comment:
Based on the documentation, should this property be part of express.CookieOptions instead of sessionOptions?

Reply:
Drats! Sure enough. Pushed a fix.